### ISSUE TITLE
Jupyter Book / MyST meetup at AGU 2024

### ISSUE THREAD

**Issue by choldgraf** [jupyter-book/team-compass#10]
Several folks in the Jupyter Book community are going to be around Washington DC this year for AGU 2024 (and related events). Let's use this as an opportunity to meet one another in person if possible!

In addition, I suspect a few folks are going to be talking at AGU and/or participating in events there. Perhaps we can use this as an opportunity to drive strategic attention to the JB/MyST projects (e.g. by asking folks to attend an informal meetup or something).

### A few things that we could do

_A few ideas that come to mind._

1. Purely social meetup with the MyST team + close friends.
2. A `Docathon` that is focused around helping communities create or upgrade content with JB 2.0 or MyST.
3. A sign-up for AGU attendees to provide user feedback for JB 2.0 and MyST
4. A development-focused working session

### Who is attending?

- @choldgraf will be there just before AGU, and could potentially stick around for a day or two to meet up with people.
- @jmunroe and @colliand will be attending (and I believe @colliand is giving a talk about co-creation)
- @rowanc1 and @stevejpurves are in WDC and I think attending (please edit/correct me if I'm wrong!)

### Rough timing plan

_Please edit the following as an agenda comes into focus_.

- **Thursday, December 12th**: Dinner in the evening
- **Friday, December 13th**: Team meet-up during the day, dinner in the evening.

### Potential discussion topics

- https://github.com/jupyter-book/mystmd/issues/1026 and related questions about any breaking changes to AST

**Comment by rowanc1** [comment]
The MyST / JB parts of the week are planning for Thursday / Friday (Dec 11 and Dec 12, 2024). We don't have a space yet, but will hopefully get somewhere close to AGU so folks can attend both.

@stevejpurves @fwkoch @agoose77 @rowanc1 will be attending! I think @choldgraf will be there for part.

cc @fperez

**Comment by fperez** [comment]
Can we do i Thursday? I'm flying back Thursday night (family calls).

**Comment by choldgraf** [comment]
@fperez I don't believe that @rowanc1 or myself can make Thursday earlier in the day, because the HHMI workshop goes until 6pm on Wednesday (and it's only a ~2 day conference so my guess is they'll go right up to the end). Perhaps you can meet up with @agoose77 or @fwkoch ?

I think the plan is to have dinner Thursday and group meetings on Friday, ah that sucks that you'll be gone :-/ .

**Comment by fperez** [comment]
No worries @choldgraf - it's a bummer but I can't really stay longer, family calls...

I'd love to meet others from the team who may be there at another time, though! @stevejpurves @fwkoch @agoose77 LMK if you folks would be open to connecting on different day!

I'm tied up for the honors banquet Wednesday starting at 8:30pm (Jessica Scheick of icepyx fame is getting the open science award!) but I'd be happy to grab a drink/tea earlier, or dinner Monday or Tuesday, or lunch any day (Mon-Thur).

**Comment by fperez** [comment]
Ah, quick note! There will be on Wednesday, 6-7pm, a session at Salon G (Convention Center) titled _Notebooks Now! What’s Next for the Computational Notebook Submission Format at AGU_ that is probably very relevant to many of us (I'll be there). Headed by Brian Sedora from AGU.
